assets/javascripts/components/alert/index.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+export const DEFAULT_SELECTOR = 'div.alert';
+
+const DEFAULT_OPTIONS = {
+    hideAfter: 5000,
```

Comment:
@Lahphim IMO 5 seconds is a bit too long, how about 3 seconds?

Reply:
updated https://github.com/Lahphim/go-crawler-challenge/pull/53/commits/054f45d16e63e515a882de350bd71a946e42aae3